examples/SingleTransmon/SingleTransmon.jl

```diff
@@ -130,7 +134,7 @@ function single_transmon(;
 
     #### Prepare solid model
     # Specify the extent of the simulation domain.
-    substrate_x = 4mm
+    substrate_x = wave_ports ? readout_length : 4mm # wave port domain boundary needs to touch the readout line
```

Comment:
I noticed that the Q I get for the wave port model is lower by over a factor of 2. I think it is basically because the absorbing boundary is closer, since I get better agreement if I have the readout length also be 4mm. Could you make the wave port option force the transmission line to be longer (=4mm) instead of making the domain shorter?